virttest/qemu_devices/qdevices.py

```diff
@@ -1504,6 +1573,36 @@ def verify_hotplug(self, out, monitor):
                 return True
         return False
 
+    def _cmdline_json(self):
+        command_dict = {}
+        out = "-%s " % self.type
+        params = self.params.copy()
+        params = self._convert_memobj_args(params)
+        command_dict["qom-type"] = params["backend"]
+        del params["backend"]
+        return out + '\'' + json.dumps(dict(command_dict, **params)) + '\''
+
+    @staticmethod
+    def _convert_memobj_args(args):
+        """
+        Convert string type of string to uint64, and create new dict
+        (e.g: "size": "14336M" ) from old dict(e.g: "size": 15032385536)
```

Comment:
Refer to your code below, I guessed you'd like to convert 14366M to 15032385536 (Bytes)